qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -39,10 +39,20 @@
 
 from .authproxy import JSONRPCException
 
+# Placeholder for better logging framework (replacing raw print statements)
+class Logger:
+    def __getattr__(self, name):
+        def p(*args, **kwargs):
+            print(args[0] % args[1:])
+        return p
+
+    def debug(*args, **kwargs):
+        pass # ignore
```

Comment:
We have converted many qa tests to use the standard python based logger.  See txPerf.py for example.  It imports "logging" and then uses logging.info(...).  I think that this code is redundant with that and should be removed.  If its better or its purpose is different, let's discuss.

Reply:
Done